### ISSUE TITLE
Improve typography of article text

### ISSUE THREAD

**Issue by Ckbhatia** [Ckbhatia/Ck-gatsby-blog#6]
Improve typography of article text.

Desired:

![desired](https://user-images.githubusercontent.com/37036881/94950542-69e9c900-0500-11eb-98c2-f54e5c0a2624.png)

Current:

![image](https://user-images.githubusercontent.com/37036881/94950744-bcc38080-0500-11eb-8bc2-af7c29db30a1.png)

Feel free to change font type if needed to match desired typography.

Note: starter has a dark mode that should be in mind.

**Comment by Ckbhatia** [comment]
Please star the project when you want to contribute to this repo

**Comment by theCypher47** [comment]
@Ckbhatia Hi, I would like to contribute in this

**Comment by Ckbhatia** [comment]
Hi @theCypher47, I am happy to see your interest to work on this. I am assigning you this.

**Comment by theCypher47** [comment]
@Ckbhatia i just want to know if there's a way to access the articles without making the whole website in local?

**Comment by Ckbhatia** [comment]
Hi, I am going to update README soon. There will be demo tokens for
articles.

**Comment by Ckbhatia** [comment]
For Demo app:
```
CONTENTFUL_SPACE_ID=zorf8a5obm1d
CONTENTFUL_ACCESS_TOKEN=v4WRlGF6fTfBUrP54Hbaiava6VjPgKbW8mFEOgzEiPY
```

You can use these

**Comment by Ckbhatia** [comment]
@theCypher47 Any update?

**Comment by Ckbhatia** [comment]
@theCypher47 Any update?

**Comment by IsaacArnold** [comment]
Hi! I'd like to work on this issue please.

**Comment by Ckbhatia** [comment]
Your can start working on it and raise a PR.

**Comment by IsaacArnold** [comment]
Thank you! 
Can you please add the current hacktoberfest label? Thanks!

**Comment by Ckbhatia** [comment]
Up-for-grabs label removed

**Comment by IsaacArnold** [comment]
Hi - unfortunately I'm unable to complete this task. Please unassign from me. Thanks!

**Comment by Mayank-Sharma17** [comment]
@Ckbhatia I would like to work on this issue; as previously stated, we need to improve the typography of the articles, which font should we use?

**Comment by Ckbhatia** [comment]
Please use font-family: `AkkuratPro, Arial, sans-serif`

**Comment by Mayank-Sharma17** [comment]
@Ckbhatia  ok, when it's done, I'll make a PR

**Comment by Mayank-Sharma17** [comment]
@Ckbhatia I am unable to build up the project; I ran `npm i` and received numerous warnings and problems. I think there is some dependency issue or issue with versions of the npm packages.

### Screenshots
![Screenshot 2023-10-14 180246](https://github.com/Ckbhatia/Ck-gatsby-blog/assets/113251342/615d680c-c063-4290-aeda-d743408dbd2e)
![Screenshot 2023-10-14 180320](https://github.com/Ckbhatia/Ck-gatsby-blog/assets/113251342/8b3aa5e2-0142-4ea5-8522-e0701749c068)
![Screenshot 2023-10-14 180337](https://github.com/Ckbhatia/Ck-gatsby-blog/assets/113251342/7ff59636-158c-47d9-b992-ca70aca38552)
![Screenshot 2023-10-14 180354](https://github.com/Ckbhatia/Ck-gatsby-blog/assets/113251342/b19e1156-00e0-46a1-9cd6-cfc66169d639)

**Comment by Ckbhatia** [comment]
Other dependencies require python2 to be installed on your system. Please install that and try again.

**Comment by Mayank-Sharma17** [comment]
Ok, I'll try again.

**Comment by Ckbhatia** [comment]
Hey @Mayank-Sharma17, is there any progress so far?

**Comment by Mayank-Sharma17** [comment]
@Ckbhatia I installed Python 2.7.0, but the errors persist. Do I need to install a specific version of Python?

**Comment by Ckbhatia** [comment]
Yes, you should have installed the specific version only if you are
installing python for this only.

**Comment by Mayank-Sharma17** [comment]
> Yes, you should have installed the specific version only if you are
installing python for this only.

which version then ?
